MAUI/Maps/tooltip.md

```diff
@@ -271,6 +271,110 @@ The [`ShowMarkerTooltip`](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Ma
 
 ## Appearance customization
 
+You can customize the below appearances of the tooltip.
+
+* **Background** - Change the background color of the tooltip in the maps using the MapTooltipSettings.Background property.
```

Comment:
* **Background** - Change the background color of the tooltip in maps using the MapTooltipSettings.Background property.